client/src/pages/Add.jsx

```diff
@@ -3,21 +3,29 @@ import Navigation from '../components/Navigation';
 import BoulderCard from '../components/BoulderCard';
 import AddClimbedBoulderForm from '../components/AddClimbedBoulderForm';
 import { useParams } from 'react-router-dom';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
-const Add = ({ bouldersList, title }) => {
+const Add = () => {
   const { id } = useParams();
-  const currentBoulder = bouldersList.find(boulder => boulder.id === id);
+  const [currentBoulder, setCurrentBoulder] = useState();
   const [newClimbedBoulder, setNewClimbedBoulder] = useState();
 
+  const fetchCurrentBoulder = () => {
+    fetch(`/api/add/${id}`)
+      .then(res => res.json())
+      .then(data => setCurrentBoulder(data));
+  };
+  useEffect(() => {
+    fetchCurrentBoulder();
+  }, []);
+
```

Comment:
It may be that you get a warning here that the `useEffect` is missing the dependency `fetchCurrentBoulder`. The easiest way to fix this is to put the function inside of the `useEffect` and not only the function call. If there is no warning, just ignore my comment. 👍

Reply:
Hi Mareike, there is no warning here. So I will ignore your comment. ;-)